src/assets/v1.3.0/sign_message_lib.json

```diff
@@ -153,8 +152,10 @@
     "73799": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "80001": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "84531": "0x98FFBBF51bb33A056B08ddf711f289936AafF717",
+    "84532": "0x98FFBBF51bb33A056B08ddf711f289936AafF717",
     "200101": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "200202": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
+    "333999": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
```

Comment:
Missing, but unable to verify. Using what the address should be based on other contracts.